meson.build

```diff
@@ -20,6 +20,7 @@ project_description = 'Implementation Guardian of Analysis Algorithms'
 # resolve dependencies
 fmt_dep  = dependency('fmt',   version: '>=9.1.0', method: 'pkg-config')
 hipo_dep = dependency('hipo4', version: '>=4.0.1', method: 'pkg-config')
+yamlcpp_dep = dependency('yaml-cpp', version: '>=0.8.0', method: 'pkg-config')
```

Comment:
Do we need minimum version 0.8.0, or does 0.7.0 work? If you're not sure, we can use the CI to test this, since Linux runners have 0.7.0.

Reply:
I'm not sure if we need version 0.8.0 probably worth testing with version 0.7.0. Thanks!

Reply:
I added a `MinVer` = "minimum required version" workflow, to test `iguana` using these minimum versions.

0.7.0-2 works, so I dropped this minimum down accordingly (its definition moved to `meson/minimum-version.sh`)